rabbitmq-tests/src/test/java/com/datastax/oss/starlight/rabbitmqtests/javaclient/functional/DefaultExchange.java

```diff
@@ -17,11 +17,14 @@
 
 import static org.junit.Assert.fail;
 
+import com.datastax.oss.starlight.rabbitmqtests.SystemTest;
 import com.datastax.oss.starlight.rabbitmqtests.javaclient.BrokerTestCase;
 import com.rabbitmq.client.AMQP;
 import java.io.IOException;
 import org.junit.Test;
+import org.junit.experimental.categories.Category;
 
+@Category(SystemTest.class)
 public class DefaultExchange extends BrokerTestCase {
```

Comment:
This is weird,
Usually surefire runs only tests classes with a name that ends with Test or something like that.
Is this running on CI?

Reply:
yes it is
https://github.com/datastax/starlight-for-rabbitmq/blob/4655227b0265b8e96a97d20a7354a7c49cb979e5/rabbitmq-tests/pom.xml#L88-L104